Makefile

```diff
@@ -51,7 +51,11 @@ build:
 
 docker: download-csm-common
 	$(eval include csm-common.mk)
-	$(BUILDER) build -t cert-csi:latest --build-arg BASEIMAGE=$(DEFAULT_BASEIMAGE) --build-arg GOIMAGE=$(DEFAULT_GOIMAGE) .
+	@echo "Building base image from $(DEFAULT_BASEIMAGE) and loading dependencies..."
+	./scripts/build_ubi_micro.sh $(DEFAULT_BASEIMAGE)
+	@echo "Base image build: SUCCESS"
+	$(eval BASEIMAGE=cert-ubimicro:latest)
```

Comment:
Why do you need to do this inside of an eval? You are not setting anything based on variables.

Reply:
I am using it it here: $(BUILDER) build -t cert-csi:latest --build-arg BASEIMAGE=$(BASEIMAGE) --build-arg GOIMAGE=$(DEFAULT_GOIMAGE) . Every repo is using it this way ..Just wanted to keep it same for every repo.